<launch>
    
    <param name="global_map_path" type="string" value="/home/ding/backup/dataset/haibo/data/0301/110/global_map.json" />
    <param name="config_path" type="string" value="/home/ding/backup/code/haibo_lt_mapping/src/json_test/json/mapping.json" />
    <param name="bag_path" type="string" value="/home/ding/backup/dataset/haibo/data/zsq/0324_110_1.bag" />
    <param name="imu_topic" type="string" value="/imu_data"/>
    <param name="lidar_topic" type="string" value="/lidar_aeb/raw_points"/>
    <param name="min_distance" type="double" value="8.0" />
    <param name="max_distance" type="double" value="70.0" />
    <param name="bag_param" type="int" value="1" />
    

    <node pkg="json_test" type="local_map_merge" name="local_map_merge" output="screen"/>

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find json_test)/rviz/test2.rviz" />
    </group>

</launch>